### PR TITLE
Address RuboCop violations that couldn't be automatically fixed

### PR DESCRIPTION
**Pull request by mokagio**
Continues the from #18839, addressing the remaining violation that couldn't be automatically fixed.

To test... read the code?

## Regression Notes

1. Potential unintended areas of impact – N.A.
2. What I did to test those areas of impact (or what existing automated tests I relied on) – N.A.
3. What automated tests I added (or what prevented me from doing so) – N.A.

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

**Comment by mokagio** [comment]
Bumping to the 20.2 milestone because I'm code freezing 20.1 and this hasn't been reviewed yet.

File: Podfile
Comment (AliSoftware) [inline Podfile]: See https://github.com/wordpress-mobile/WordPress-iOS/pull/18847#discussion_r897909868

**Review by AliSoftware — COMMENTED** [review]
I left most of my feedback about the changes seen here in [this other PR 18847](https://github.com/wordpress-mobile/WordPress-iOS/pull/18847) which does the same and more.

**Comment by mokagio** [comment]
@AliSoftware thank you for the suggestions, can you have another look? Thanks!

I regret not realizing that I messed up the PRs stacking between this and https://github.com/wordpress-mobile/WordPress-iOS/pull/18847. #18847 should only have had the Danger-related commits 😞 

Note that I disabled auto-merge intentionally because I want to rebase to get rid of the commit on this branch before merging on `trunk`.

File: fastlane/lanes/localization.rb
Comment (AliSoftware) [inline fastlane/lanes/localization.rb]: This seems to be at the wrong place. The corresponding `rubocop:disable` on line 302 is at the same level as the function declaration (indentation level 1), while this balancing one is inside the function body (indentation level 2).

We should either move the `disable` on line 305 inside the function body, or move the `enable` on line 353 after the `end` of the function body.

**Review by AliSoftware — APPROVED** [review]
Left only one nitpick about `rubocop:enable` location, otherwise LGTM.

Approving to unblock you, as you disabled auto-merge and I trust you'll fix this nitpick before your rebase and merge 🙂

File: fastlane/lanes/localization.rb
Comment (mokagio) [reply fastlane/lanes/localization.rb]: Good catch, thanks! Clearly went for the wrong `end` 🤦‍♂️ Will move to the end of the method.

**Pull request merged**
